scripts/bamboo/build_and_test.sh

```diff
@@ -40,7 +40,7 @@ trycmd "make VERBOSE=1 -j"
 #
 echo "Testing..."
 if [[ $HOSTNAME == *manta* ]] || [[ $HOSTNAME == *ansel* ]]; then
-  lalloc 1 lrun -n 1 ctest --output-on-failure -T Test
```

Comment:
I'm (too easily) confused again.  Why didn't this work?

Reply:
Let me email you.

Reply:
If this is for CORAL machines, you should do this as lalloc 1 lrun -1 ...  i.e., not '-n 1'

Reply:
We need to avoid `lrun` entirely.

Reply:
Then, you have to submit a batch job I think.

Reply:
Yup, that's what `lalloc` is for. I've got it figured out and tested this already 😄

Reply:
Actually, the manta and ansel systems are different in terms of how you go from a login node to a compute node. ansel has an extra step involving a launch node.

Reply:
Not anymore.